libsolutil/Whiskers.cpp

```diff
@@ -36,10 +36,16 @@ Whiskers::Whiskers(string _template):
 {
 }
 
+void Whiskers::setTemplate(string _template)
```

Comment:
Why do we need this function?

Reply:
I just removed this function now, I didn't realised that `operator=` can be used.